### DISCUSSION TITLE
MIME type Error

### DISCUSSION THREAD

**Discussion by Chauban** [tailwindlabs/tailwindcss#11855] [Help]
I am new to frontend dev. I followed the CLI usage on tailwindcss official web but the chrome console get this error:

Refused to apply style from 'http://localhost:8080/dist/output.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

The web also has no css application. How to go on please?

**Comment by wongjn** [comment]
It seems like the URL is incorrect and is serving a 404 HTML response, hence the error and why no style is applied to the webpage. Consider correcting the path to the CSS file.
